concent_api/core/payments/base.py

```diff
@@ -17,50 +17,89 @@ def wrapper(*args, **kwargs):
 
 
 @_add_backend
-def sum_payments(backend, list_of_payments):
-    return backend.sum_payments(list_of_payments)
+def sum_payments(backend, list_of_payments = None):
+    return backend.sum_payments(list_of_payments = list_of_payments)
 
 
 @_add_backend
-def get_client_eth_account(backend):
-    return backend.get_client_eth_account()
+def sum_subtask_price(backend, subtask_results_accepted_list = None):
+    return backend.sum_subtask_price(subtask_results_accepted_list = subtask_results_accepted_list)
 
 
 @_add_backend
-def get_number_of_eth_block(backend, request = None):
-    return backend.get_number_of_eth_block(request)
+def get_clients_eth_accounts(backend, task_to_compute = None):
+    return backend.get_clients_eth_accounts(task_to_compute = task_to_compute)
 
 
 @_add_backend
-def payment_summary(backend, request = None, subtask_results_accepted_list = None, list_of_transactions = None, list_of_forced_payments = None):
-    return backend.payment_summary(request, subtask_results_accepted_list, list_of_transactions, list_of_forced_payments)
+def get_number_of_eth_block(backend, concent_rpc = None):
+    return backend.get_number_of_eth_block(concent_rpc = concent_rpc)
 
 
 @_add_backend
-def get_list_of_transactions(backend, _oldest_payments_ts = None, current_time = None, _to_block = None, _payer_address = None, _payee_address = None, request = None):
-    return backend.get_list_of_transactions(_oldest_payments_ts, current_time, _to_block, _payer_address, _payee_address, request)
+def sum_amount_price_for_provider(
```

Comment:
If possible, try to add typing notations everywhere when it is known. It would help a lot when working with undocumented APIs